week-3/implement/is-proper-fraction.js

```diff
@@ -32,3 +32,66 @@
 // target output: false
 // Explanation: The fraction 3/3 is not a proper fraction because the numerator is equal to the denominator. The function should return false.
 // These acceptance criteria cover a range of scenarios to ensure that the isProperFraction function handles both proper and improper fractions correctly and handles potential errors such as a zero denominator.
+
+function isProperFraction(numerator, denominator) {
+    if(denominator === 0){
+        return "Error";
+    }
+    else if(numerator > denominator){
+        return "false";
+    }
+    else if(numerator < denominator) {
+      return "true";
+    } 
+    else if(numerator < 0 && denominator > 0) {
+      return "true";
+    } 
+    else if(numerator === denominator) {
+      return "false";
+    }
+}
```

Comment:
A nice and clean implementation of the requested function. Great job.

Reply:
thank you